tutorial/20_recipes/010_reuse_best_trial.py

```diff
@@ -92,3 +92,8 @@ def detailed_objective(trial):
 # and behaves differently from :class:`~optuna.trial.Trial` in some situations.
 # For example, pruning does not work because :class:`~optuna.trial.FrozenTrial.should_prune`
 # always returns ``False``.
+#
+# .. note::
+#     For multi-objective optimization as demonstrated by :ref:`multi_objective`,
+# :func:`~optuna.study.Study.best_trials` returns a list of `:class:`~optuna.trial.FrozenTrial`
+# on Pareto front. So we can re-use each trial in the list by the similar way above.
```

Comment:
```suggestion
    # :func:`~optuna.study.Study.best_trials` returns a list of :class:`~optuna.trial.FrozenTrial`
    # on Pareto front. So we can re-use each trial in the list by the similar way above.
```

Reply:
Thanks, fixed it.
![Screenshot from 2022-03-24 21-53-55](https://user-images.githubusercontent.com/53843818/159963642-649eeb55-92f2-4839-abd0-57814fb62d12.png)